ui/lib/src/chat/interfaces.ts

```diff
@@ -18,16 +18,16 @@ export interface ChatOpts {
   noteId?: string;
   noteText?: string;
   plugin?: ChatPlugin;
-  alwaysEnabled: boolean;
   kidMode: boolean;
 }
 
-export interface ChatPlugin {
-  tab: {
-    key: string;
-    name: string;
-  };
+export type Tab = { key: string; hidden?: boolean };
+
+export interface ChatPlugin extends Tab {
+  name: string;
   view(): VNode;
+  kidSafe?: boolean; // default false
```

Comment:
curious about why we don't (seem to) load tournament standing plugin in kid mode.

Reply:
No reason I can think of. It's probably an unintended effect of disabling the chat for kids.